extras/kms/migrations/postgres/04_keys.up.sql

```diff
@@ -0,0 +1,114 @@
+begin;
+
+create table kms_root_key (
+  private_id kms_private_id primary key,
+  scope_id kms_scope_id not null unique, -- there can only be one root key for a scope.
```

Comment:
in boundary this is a foreign key with `on delete cascade`, how would that relationship be managed with this module?

Reply:
I need to document this in the README.  I'm thinking that users will likely need to add a foreign key constraint that prevents orphans, but since we have no way of determining the table name for scopes, it will be left to the user to add the constraint with guidance from our documentation.  Happy to hear other suggestions.

Reply:
The only thought that is coming to mind (and I havnt thought it through at all) would be to introduce some kind of templating.